### DISCUSSION TITLE
How to set a default sound output device?

### DISCUSSION THREAD

**Discussion by PeterKriegtNenWolf** [Vanilla-OS/live-iso#378] [Q&A]
Hello,

I would like to be able to set a sound output device as default, so when i restart the computer, that device is used by default, if available. In the settings-menu I can choose the output device but can't set it as my default.
Is there a command I can use in the terminal, that can achieve this? I tried following this guide (https://forums.debian.net/viewtopic.php?t=141811) but it seems like the command is not found by the shell. Any ideas which commands I can use?

**Comment by GabsEdits** [comment]
Add `host-shell` before all the commands
